docs/src/pages/components/data-grid/editing/FullFeaturedCrudGrid.tsx

```diff
@@ -10,14 +10,14 @@ import {
   GridRowsProp,
   useGridApiRef,
   DataGridPro,
-  GridApiRef,
   GridColumns,
   GridRowParams,
   MuiEvent,
   GridToolbarContainer,
   GridActionsCellItem,
   GridEventListener,
   GridEvents,
+  GridApiRefPro,
```

Comment:
Ideally it would be nice to keep it named as `GridApiRef` (same for `GridState`) and each package exposes a different interface. However, we can't yet because of the `_modules_` folder. Should we still use `GridApiRef` in the demos and mark `GridApiRefPro` as internal until we remove the modules folder? I guess we don't need the "Pro" suffix once each interface is moved to its correct folder.

Reply:
Right now we have `GridApiRef` which takes the api as a generic and return it's ref version. And `GridApiRefPro` / `GridApiRefCommunity` which are the plan-based apis.
This allow to reduce the typing breaking change.

But if we are looking forward to what would be ideal in v6.  ​Differentiating the two makes it easier to create a different doc page for the free and the pro interface.
If we have the same name, we could still do it (or a single page with a flag "pro only" when the property is only available on pro.

Reply:
If we add the suffix I assume we will remove `GridApiRef` in the next major, right? If we do, we have the [selectors](https://mui.com/components/data-grid/state/#catalog-of-selectors) page which uses `GridApiRef`. If we want to type them correctly, this page and all the remaining mentions should be updated in future.

> ​Differentiating the two makes it easier to create a different doc page for the free and the pro interface.

I guess each package should have its own docs, so each `GridApi` would have its own URL.

Reply:
Yes my goal was to remove `GridApiRef` in v6 as it is just an alias for `React.MutableRefObject` with a constraint on the generic.